styles.css

```diff
@@ -279,99 +386,157 @@ h2 {
   display: flex;
   flex-wrap: wrap;
   justify-content: center;
-  row-gap: 10px;
-  column-gap: 8px;
+  row-gap: 1rem;
+  column-gap: 0.8rem;
   list-style: none;
 }
 
 .skill-list li {
-  font-size: 12px;
+  font-size: 1.2rem;
   font-style: normal;
   font-weight: 600;
-  line-height: 16px;
+  line-height: 1.6rem;
   text-align: center;
   color: #36b37e;
   background: #fff;
-  border-radius: 8px;
-  padding: 12px;
+  border-radius: 0.8rem;
+  padding: 1.2rem;
 }
 
 .chat {
   display: flex;
   justify-content: center;
-  margin: 55px auto;
+  margin: 5.5rem auto;
 }
 
 .chat h2 {
   font-style: normal;
   font-weight: 700;
-  font-size: 40px;
-  line-height: 52px;
+  font-size: 4rem;
+  line-height: 5.2rem;
   text-align: center;
   color: #172b4d;
-  width: 327px;
+  width: 70%;
 }
 
 #name,
 #email {
-  width: 327px;
-  border: 1px solid #d0d9d4;
-  border-radius: 4px;
+  width: 70%;
+  border: 0.1rem solid #d0d9d4;
+  border-radius: 0.4rem;
   font-style: normal;
   font-weight: 400;
-  font-size: 15px;
-  line-height: 24px;
+  font-size: 1.5rem;
+  line-height: 2.4rem;
   color: #091e42;
-  padding: 15px 16px 15px 16px;
+  padding: 1.5rem 1.6rem 1.5rem 1.6rem;
 }
 
 #textarea {
-  width: 327px;
-  border: 1px solid #d0d9d4;
-  border-radius: 4px;
-  height: 114px;
+  width: 70%;
+  border: 0.1rem solid #d0d9d4;
+  border-radius: 0.4rem;
+  height: 11.4rem;
   font-style: normal;
   font-weight: 400;
-  font-size: 15px;
-  line-height: 24px;
+  font-size: 1.5rem;
+  line-height: 2.4rem;
   color: #091e42;
-  padding: 12px;
+  padding: 1.2rem;
+  resize: none;
 }
 
-footer {
-  background-image: url(/images/illustration-contact-form.svg);
+.footer-page {
+  display: grid;
+  grid-template-columns: 1fr;
+  background-image: url(./images/illustration-contact-form.svg);
   background-position: 100% 78%;
   background-repeat: no-repeat;
 }
 
+@media screen and (min-width: 768px) {
+  .chat {
+    flex-direction: column;
+    justify-content: start;
+    margin: 3rem auto;
+  }
+
+  main {
+    background-image:
+      url(./images/illustration-1-contact-form-desktop.svg),
+      url(./images/illustration-2-contact-form-desktop.svg),
+      url(./images/contact-form-illustratoin-3-desktop.svg);
+    background-position: 0% 100%, 20% 97%, 100% 100%;
+    background-repeat: no-repeat;
+  }
+
+  .footer-page {
+    grid-template-columns: 2fr 3fr;
+    column-gap: 2.4rem;
+    background-image: none;
+    margin: 16.4rem 6.5% auto 6.5%;
+  }
+
+  #name,
+  #email,
+  #textarea {
+    width: 100%;
+  }
+
+  .chat h2 {
```

Comment:
- [ ] [OPTIONAL] Kindly make sure your `text` in the `contact me` `section` is aligned and looks as close as possible to the `Figma` design 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/193285843-2a5d57c2-cb14-4d82-b4c5-b95adaa1bfe3.png)  |  ![image](https://user-images.githubusercontent.com/74814780/193285761-f3a4ea87-c242-49c0-aaa5-327e9d943fe4.png)